### ISSUE TITLE
Wrong pins on CAM1 connector for i2c

### ISSUE THREAD

**Issue by manmountain** [Blinkinlabs/RaspberryPincushion#1]
I might have spotted an error in the compute module HDMI out.SchDoc

I think the wrong pins are used for  PI_SDA1 PI_SCL0. The i2c pins used in the schematic is for CAM0 and not CAM1.

It would be useful if the DSI1 connector  was added to he board also

**Comment by cibomahto** [comment]
Thanks! I'll take a look in a bit.

Are you interested in the board? Our original manufacturing plans fell through, and we haven't put together a new one yet.

**Comment by manmountain** [comment]
Im interested in the board. Seems fun :)

Do you think its possible to shrink it abit? Its alot cheaper to make a 100x100mm board then a 140x90mm ;)

**Comment by cibomahto** [comment]
Yeah, that's crossed my mind as well. It seems possible, but I'd like to find some funding for it before I put in too much more work. Possibly after we get our CM4-based router design done.
